makefile.shade

```diff
@@ -1,20 +1,18 @@
 use namespace="System.IO"
 
--BuildQuality = "preview4";
+- //BuildQuality = "preview4";
 use-standard-lifecycle
 k-standard-goals
 
 #repo-initialize target='initialize'
     use-volatile-feed
-    @{
-      var directoriesToRestore = "test";
-      if (Directory.Exists("src"))
-      {
-        directoriesToRestore = "src " + directoriesToRestore;
-      }
-    }
-    dotnet command='restore ${directoriesToRestore} ${E("KOREBUILD_DOTNET_RESTORE_OPTIONS")}' workingDir='${Directory.GetCurrentDirectory()}' if='!NoRestore'
+    dotnet command='restore Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.csproj ${E("KOREBUILD_DOTNET_RESTORE_OPTIONS")}' dotnetDir='${Path.Combine(Directory.GetCurrentDirectory(), "src", "Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design")}' if='!NoRestore'
+    dotnet command='restore Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.Test.csproj ${E("KOREBUILD_DOTNET_RESTORE_OPTIONS")}' dotnetDir='${Path.Combine(Directory.GetCurrentDirectory(), "test", "Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.Test")}' if='!NoRestore'
+    dotnet command='restore Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests.csproj ${E("KOREBUILD_DOTNET_RESTORE_OPTIONS")}' dotnetDir='${Path.Combine(Directory.GetCurrentDirectory(), "test", "Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests")}' if='!NoRestore'
     - Directory.CreateDirectory(Path.Combine(Directory.GetCurrentDirectory(), "artifacts", "build"));
 
 #xunit-test
-    - // Temporarily no-op until we have a resolution for https://github.com/aspnet/MvcPrecompilation/issues/48
\ No newline at end of file
+    - // Temporarily no-op until we have a resolution for https://github.com/aspnet/MvcPrecompilation/issues/48
```

Comment:
This should no longer be an issue, so we should remove this and start running tests.